Assets/Scripts/DogAI/DogAI.cs

```diff
@@ -0,0 +1,116 @@
+﻿/*
+ * Author(s): Timothy Ng
+ * Description: Brains behind the dogs actions on the world
+ * Usage: [no notes]
+ */
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using k = PPGlobal;
+
+public class DogAI : MonoBehaviour {
```

Comment:
- Why is this a separate class from `Dog`?
- Should inherit from `MonoBehaviourExtended` (default if no more specific class exists)
- Should save a reference to which dog it is.